data/hai/hai.txt

```diff
@@ -618,6 +618,60 @@ phrase "friendly unfettered"
 	word
 		" Hai."
 
+phrase "hai bribe acceptance hail"
+	word
+		"I"
+		"We"
+		"The Hai"
+	word
+		"feel"
+		"think that it is"
+	word
+		"shameful"
+		"vile"
+	word
+		"to work with you"
```

Comment:
Remember that phrases don't put spaces between words automatically. This also applies to the other Hai hail.